tests/acceptance/new-test.js

```diff
@@ -285,6 +287,14 @@ describe('Acceptance: ember new', function() {
   }));
 
   it('ember new uses yarn when blueprint has yarn.lock', co.wrap(function *() {
+    try {
+      let yarn = new Command('yarn');
+      yarn.invoke();
+    } catch (err) {
+      console.log('No yarn present, skipping test');
+      return;
```

Comment:
instead of returning from the function we should properly mark it as "skipped". if I remember correctly there are two ways of doing it:
a) calling `this.skip()` in the test code
b) using `it.skip('foo', function() {...})` instead of `it(...)`